common/DiffMatchPatch.java

```diff
@@ -146,19 +128,19 @@ protected LinesToCharsResult(String chars1, String chars2,
   }
 
   /**
-   * Find the differences between two texts.  Simplifies the problem by
-   * stripping any common prefix or suffix off the texts before diffing.
+   * Find the differences between two texts. Simplifies the problem by stripping any common prefix
+   * or suffix off the texts before diffing.
+   *
    * @param text1 Old string to be diffed.
    * @param text2 New string to be diffed.
-   * @param checklines Speedup flag.  If false, then don't run a
-   *     line-level diff first to identify the changed areas.
-   *     If true, then run a faster slightly less optimal diff.
-   * @param deadline Time when the diff should be complete by.  Used
-   *     internally for recursive calls.  Users should set DiffTimeout instead.
+   * @param checklines Speedup flag. If false, then don't run a line-level diff first to identify
+   *     the changed areas. If true, then run a faster slightly less optimal diff.
+   * @param deadline Time when the diff should be complete by. Used internally for recursive calls.
+   *     Users should set DiffTimeout instead.
    * @return Linked List of Diff objects.
    */
-  private LinkedList<Diff> diff_main(String text1, String text2,
-                                     boolean checklines, long deadline) {
+  private LinkedList<Diff> diff_main(
+      String text1, String text2, boolean checklines, long deadline) {
```

Comment:
The explicit type argument `<Diff>` (`diffs = new LinkedList<Diff>()`) can be replaced with `<>` (line 152). 
Also below.